GetStream/stream-chat-swift
# [CIS-2209] Fix message text not dynamically scalable with content size category changes

### 🔗 Issue Links
CIS-2209
https://github.com/GetStream/stream-chat-swift/issues/2325

### 🎯 Goal
Fix message text not dynamically scalable with content size category changes.

### 🛠 Implementation

_Provide a detailed description of the implementation and explain your decisions if you find them relevant._

### 🎨 Showcase
<img src="https://user-images.githubusercontent.com/12814114/194657627-fd0c54dc-1636-463b-a92d-6456c52a33cd.gif" width="30%"/>

### 🧪 Manual Testing Notes
- Open a Channel
- Open Xcode > Developer Tools > Accessibility Inspector
- Select the Simulator
- Select the 3rd top right corner button
- Change the dynamic type
- Should change the message content text

### ☑️ Contributor Checklist
